guides/v2.2/javascript-dev-guide/javascript/js_mixins.md

```diff
@@ -105,6 +105,29 @@ define(['jquery'], function ($) {
 });
 ```
 
+#### Extend JS Object
+
+The last kind of JS mixin is when the base Javscript file returns an object. In this case, a wrapper is necessary. The following is an example of a mixin that extends the `setHash` function of [step navigator object][]. Here, `_super()` is the base method which can be called if needed.
```

Comment:
To make this statement more future-proof, let's adjust the description slightly:

```suggestion
Another use-case of JS mixin is when the base Javascript file returns an object. In this case, a wrapper is necessary. The following is an example of a mixin that extends the `setHash` function of [step navigator object][]. Here, `_super()` is the base method that can be called if needed.
```

Thank you!